src/geojson/geojson.ml

```diff
@@ -290,13 +291,16 @@ module Make (J : Intf.Json) = struct
           match J.to_string typ with
           | Ok "Feature" -> (
               match
-                (J.find json [ "geometry" ], J.find json [ "properties" ])
+                ( J.find json [ "geometry" ],
+                  J.find json [ "properties" ],
+                  J.find json [ "foreign_members" ] )
               with
-              | Some geometry, props ->
+              | Some geometry, props, forgn ->
                   Result.map
```

Comment:
The type of `Result.map` is 

```ocaml
val map : ('a -> 'b) -> ('a, 'e) result -> ('b, 'e) result
```

The first argument is a function that "maps" from one value to another if the result is an `Ok v`. The `map` is used here because parsing the geometry field might produce an error, in which case we want to return that. Otherwise we wrap it in an option and return the pair for a geometry and the properties (which is also optional). The properties are very similar to how the foreign members should be treated. 

Note, however, there is no JSON field called `"foreign_members"`. The `"foreign_members"` constitute anything else that may be left in the object after trying to parse it.

This section of RWO might help with the error handling https://dev.realworldocaml.org/error-handling.html and the example of a foreign member in the RFC is equally useful https://datatracker.ietf.org/doc/html/rfc7946#section-6.1